guest-os-info.md

```diff
@@ -0,0 +1,35 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    os-short-id:
+```
+
+The os-short-id label is based on the short OS identifier from [libosinfo](https://libosinfo.org/).
+Libosinfo database can be queried to extract additional information about the operating system using the short-id:
+
+```
+# List all operating systems
+$ osinfo-query os
+
+
+ Short ID             | Name                            | Version | ID
+---------------------------------------------------------------------------------------------------
+...
+ fedora24             | Fedora 24                    | 24        | http://fedoraproject.org/fedora/24
+ fedora25             | Fedora 25                    | 25        | http://fedoraproject.org/fedora/25
+ fedora26             | Fedora 26                    | 26        | http://fedoraproject.org/fedora/26
+...
+
+```
+
```

Comment:
You might want to add that there are additional fields to query, man osinfo-query will help to find them.

yabe alo a note that there are xml files with the same informations? but maybe this is for the developer guide.